pkg/operator/k8sutil/node.go

```diff
@@ -97,6 +98,17 @@ func GetValidNodes(ctx context.Context, rookStorage cephv1.StorageScopeSpec, cli
 	return RookNodesMatchingKubernetesNodes(rookStorage, validK8sNodes)
 }
 
+func NodeWithHostnameExists(ctx context.Context, clientset kubernetes.Interface, hostName string) (bool, error) {
+	options := metav1.ListOptions{LabelSelector: fmt.Sprintf("%s=%s", LabelHostname(), hostName)}
+	nodes, err := clientset.CoreV1().Nodes().List(ctx, options)
+	if err != nil {
+		logger.Errorf("error querying node %q. %v", hostName, err)
+		return false, errors.Wrapf(err, "failed to query node %q", hostName)
```

Comment:
I'm confused. You said you'd remove the log here: https://github.com/rook/rook/pull/16249#discussion_r2257848481

Do you think there is reason to keep the log as well as the return?

Reply:
Bah, I fully intended to remove the log statement and only return the error. Pushing it now...